### PR TITLE
Adding Windows builds to CI

### PR DESCRIPTION
**Pull request by iSynaptic**

### CODE REVIEW

**Comment by iSynaptic** [comment]
@r8d8 Should address remaining concerns and close #156.

**Comment by aleksuss** [comment]
@iSynaptic LGTM but there is an error while building on windows. `geteuid()` is absent on windows platform. 
https://travis-ci.org/rust-rocksdb/rust-rocksdb/jobs/451362677#L180

**Comment by iSynaptic** [comment]
@aleksuss Yeah, I'm working on that. Created the PR to give visibility to this work in-progress. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -26,3 +27,5 @@ script:
   - cargo test
 
 cache: cargo
+before_cache:
+  - rm -rfv target/
```
Comment (aleksuss) [inline .travis.yml]: @iSynaptic LGTM but add a new line here ?

Comment (iSynaptic) [reply .travis.yml]: Done.

**Pull request merged**